src/userservice/userservice.py

```diff
@@ -217,11 +217,31 @@ def _shutdown():
         """Executed when web app is terminated."""
         app.logger.info("Stopping userservice.")
 
-    # Set up logger
-    app.logger.handlers = logging.getLogger('gunicorn.error').handlers
-    app.logger.setLevel(logging.getLogger('gunicorn.error').level)
-    app.logger.info('Starting userservice.')
-
+    # set log formatting
+    date_format = "%Y-%m-%d %H:%M:%S"
+    message_format = '%(asctime)s | [%(levelname)s] | %(funcName)s | %(message)s'
+    logging.basicConfig(format= message_format, datefmt= date_format, stream=sys.stdout)
+
+    # set log level
+    log_levels = {
```

Comment:
This way of setting the log level does seem clunky to me. Before, we let gunicorn parse the user-provided log level string, seems uwsgi doesn't support that.